src/youtube-transcript/index.ts

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env node
+
+import { Server } from "@modelcontextprotocol/sdk/server/index.js";
+import { StdioServerTransport } from "@modelcontextprotocol/sdk/server/stdio.js";
+import {
+  CallToolRequestSchema,
+  ListToolsRequestSchema,
+  ErrorCode,
+  McpError,
+  Tool,
+  CallToolResult,
+} from "@modelcontextprotocol/sdk/types.js";
+// @ts-ignore
+import { getSubtitles } from 'youtube-captions-scraper';
```

Comment:
Would be nice to use Youtube Data API V3 for management of the YouTube channel / automated posting.

This API for getting captions is not really official.

Reply:
> Would be nice to use Youtube Data API V3 for management of the YouTube channel / automated posting.
> 
> This API for getting captions is not really official.


Good idea. I will check Youtube Data API V3. I will modify it so that it can be changed to the corresponding API. Thank you.